utils/sites.js

```diff
@@ -242,9 +242,15 @@ const updateParticipantData = async (req, res, authObj) => {
         // Reject to update the uninvited flag if the participant is verified or The PIN was used to sign in
         const uninvitedRecruitsId = fieldMapping.participantMap.uninvitedRecruits.toString()
         if (dataObj[uninvitedRecruitsId] && dataObj[uninvitedRecruitsId] === fieldMapping.yes &&
-            (docData[fieldMapping.verified.toString()] === fieldMapping.verified || docData['Connect_ID'])) {
+            (
+                docData[fieldMapping.participantMap.signedInFlag.toString()] === fieldMapping.yes
```

Comment:
It looks like you're converting the signedInFlag toString for a triple equality comparison with fieldMapping.yes and fieldMapping.verified, but aren't those constants both numbers? Should probably convert them both to strings for this comparison to be safe.

Reply:
Scratch that, misread this.